DatadogWebViewTracking/Sources/WebViewTracking.swift

```diff
@@ -0,0 +1,57 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import DatadogInternal
+
+/// Defines methods to send WebView related information
+public protocol WebViewTracking {
+    /// Sends a bag of data
+    /// - Parameter body: The data to send, it must be parsable to `WebViewTrackingMessage`
+    func send(body: Any) throws
+
+    /// Sends a message
+    /// - Parameter message: The message to send
+    func send(message: WebViewTrackingMessage) throws
+}
```

Comment:
For conformance it would only need the `send(message:)` method, while the convenient `send(body:)` could be provided in an extension:
```swift
/// Defines methods to send WebView related information
public protocol WebViewTracking {
    /// Sends a message
    /// - Parameter message: The message to send
    func send(message: WebViewTrackingMessage) throws
}

extension WebViewTracking {
    /// Sends a bag of data to the message bus
    /// - Parameter body: The data to send, it must be parsable to `WebViewTrackingMessage`
    public func send(body: Any) throws {
        let message = try WebViewTrackingMessage(body: body)
        try send(message: message)
    }
}
```

Reply:
Swifty, done.